src/frontends/paddle/src/op/gather.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs gather(const NodeContext& node) {
+    const auto data_node = node.get_input("X");
+    const auto index_node = node.get_input("Index");
+    Output<Node> axis_node;
+
+    PADDLE_OP_CHECK(node, !node.has_input("Axis"), "OpenVINO not support parameterized Axis input!");
```

Comment:
Why can't we support parametrized axis? ov::Gather accepts axis as input, not attribute.

Reply:
yes, "ov::Gather accepts axis as input," but we found only constant operation on 'axis' input is supported(https://github.com/openvinotoolkit/openvino/blob/9bc7ebda7b4fee67f3c27dff18581a6f51f4c098/src/plugins/intel_cpu/src/nodes/gather.cpp#L34). and axis as input of Paddle gather op is not constant.
because these limitation doesn't block our current target models, so do you think we would better keep these frontend limitation until plugin fix it?

Reply:
I think this internal limitation is not something that frontend should be blocked by. I would recommend implementing this optional input and have tests for it disabled or marked as xfail, and have a ticket for the plugin to fix that. But this is more like a nice-to-have thing, not a blocker for this PR.

Reply:
implemented 'axis' input for 'Gather' Op conversion with testcase comment(CVS-82724 is created to track it).